### ISSUE TITLE
Explore section card amount in each row

### ISSUE THREAD

**Issue by RafeeDaBoy** [N0-0NE-Dev/NoFasel#39]
**Describe the bug**
From the app screenshots and issue #33 I think there's supposed to be 2 cards in each row. 

But when going to the explore section, there's 1 card in each row and 8 cards in the whole column.

I really love how the app looks, but wasn't sure if this was the intended design.

**To Reproduce**
1. Go to 'Explore'
2. See error

**Expected behavior**
2 cards in each row.

**Screenshots**
![Screenshot_2023-03-06-21-32-44-63_04aad770f80d7e5bbd6c1fff15b6a3f5__01](https://user-images.githubusercontent.com/69535896/223158920-bc0bcfa0-4689-4843-85bf-2aae6061ce89.jpg)

**Smartphone (please complete the following information):**
 - Device: OnePlus 9RT
 - OS: Android 11
 - Version 1.1.7

**Comment by DarkCrypt** [comment]
I agree with this. There should be 2 cards per row when device is vertical and more than 8 total cards in explore section. When I turn my device horizontal there's an appropriate 4 cards per row.
